src/Modifier/RemoveKeysFromVendor.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Config\Modifier;
+
+use InvalidArgumentException;
+
+/**
+ * @see RemoveFromVendor::keys()
+ */
+final class RemoveKeysFromVendor
```

Comment:
Maybe mark as internal?

Reply:
`RemoveFromVendor::keys()` returned this to userland... I do not know if it is necessary to do internal.

Reply:
OK